jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -237,26 +245,44 @@ def vue_do_aper_phot(self, *args, **kwargs):
                     d['id'] = 1
                     self.app._aper_phot_results = _qtable_from_dict(d)
 
-            # Radial profile
             reg_bb = reg.bounding_box
             reg_ogrid = np.ogrid[reg_bb.iymin:reg_bb.iymax, reg_bb.ixmin:reg_bb.ixmax]
             radial_dx = reg_ogrid[1] - reg.center.x
             radial_dy = reg_ogrid[0] - reg.center.y
             radial_r = np.hypot(radial_dx, radial_dy).ravel()  # pix
             radial_img = comp_no_bg_cutout.ravel()
-            if comp.units:
-                y_data = radial_img.value
-                y_label = radial_img.unit.to_string()
-            else:
-                y_data = radial_img
-                y_label = 'Value'
+
+            # Radial profile (Raw)
+            if self.current_plot_type == "Radial Profile (Raw)":
+
+                if comp.units:
+                    y_data = radial_img.value
+                    y_label = radial_img.unit.to_string()
+                else:
+                    y_data = radial_img
+                    y_label = 'Value'
+                x_data = radial_r
+                x_label = "pix"
+
+            # Radial Profile
+            elif self.current_plot_type == "Radial Profile":
+                flux = np.bincount(list(radial_r), radial_img)

```

Comment:
Is the casting to `list` necessary here?

Reply:
Surprisingly yes! This line throws an error without it, something about a `typeerror` between 64 bit and 32 bit numbers.